tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrderingPartyControllerTests.cs

```diff
@@ -49,20 +48,20 @@ public static void Constructor_VerifyGuardClauses()
         [Test]
         [InMemoryDbAutoData]
         public static async Task GetAsync_OrderExists_ReturnsTheOrderingParty(
-            [Frozen] ApplicationDbContext context,
+            [Frozen] Mock<IOrderingPartyService> service,
             [Frozen] CallOffId callOffId,
             Order order,
             OrderingPartyController controller)
         {
             order.OrderingParty.Should().NotBeNull();
             order.OrderingPartyContact.Should().NotBeNull();
 
-            context.Add(order);
-            await context.SaveChangesAsync();
+            service.Setup(o => o.GetOrder(callOffId)).ReturnsAsync(order);
 
             var expectedValue = new OrderingPartyModel(order.OrderingParty, order.OrderingPartyContact);
 
             var response = await controller.GetAsync(callOffId);
+            service.Verify(o => o.GetOrder(callOffId));
```

Comment:
I don't think we need to `Verify` here as the test covers this as part of its assertion.

Reply:
Removed `service.Verify()`